core/storage/user/gae_models.py

```diff
@@ -1540,14 +1662,17 @@ def create(cls, user_id, exploration_id):
         return cls(
             id=instance_id, user_id=user_id, exploration_id=exploration_id)
 
+    # We have ignored [override] here because the signature of this method
+    # doesn't match with BaseModel.get().
     @classmethod
-    def get(cls, user_id, exploration_id):
+    def get(cls, user_id, exploration_id): # type: ignore[override]
+        # type: (Text, Text) -> Optional[ExplorationUserDataModel]
         """Gets the ExplorationUserDataModel for the given user and exploration
          ids.
 
         Args:
             user_id: str. The id of the user.
-            exploration_id: str. The id of the exploration.
+            exploration_id: str|None. The id of the exploration.
```

Comment:
Why is this changed?

Reply:
By mistake, wanted to add None to its return type. Thanks!!